src/lib/markbind/src/parsers/componentParser.js

```diff
@@ -42,6 +42,26 @@ function _parseAttributeWithoutOverride(element, attribute, isInline, slotName =
   delete el.attribs[attribute];
 }
 
+/**
+ * Takes an element, looks for direct elements with slots and transforms to avoid Vue parsing.
+ * @param element Element to transform
+ */
+function _transformSlottedComponents(element) {
+  element.children.forEach((c) => {
+    const child = c;
```

Comment:
Might be missing something here, but could we just rename the `c` parameter to `child` directly instead? 😅

Reply:
Yeah, It's because of our eslint rule `no-param-reassign`